locale/locales/en.json

```
@@ -31,6 +31,14 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.RowsPerPageChange": {
+    "value": "Rows per page has changed. Now showing {0} rows on each page.",
```

Maybe we should say that you go back to the first page, since you always do that so something like
```suggestion
    "value": "Rows per page has changed. Now showing {0} rows on each page and moved to first page.",
```

---

you mean "the first row" right?

---

✅ fixed